### PR TITLE
Onboarding flow implemented in storybook

### LINKED ISSUE #84: Implement welcome page

### LINKED ISSUE #85: Implement terms page
- if the user clicks on "accept & participate", some event is fired from the terms page component.
- if the user clicks on "cancel", another event is fired from the terms page component.

### LINKED ISSUE #86: Implement demographics page

### PR DESCRIPTION
**Pull request by hamilton**
closes #84, closes #85, closes #86 

I found that a branch per page was getting a bit cumbersome, so this branch builds off of the commits for the welcome / TOS / demographics pages.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -10,6 +10,7 @@
     <link rel="stylesheet" href="common.css" />
     <link rel="stylesheet" href="pioneer.css" />
     <link rel="stylesheet" href="global.css" />
+    <!-- <link rel="stylesheet" href="build/external.css" /> -->
```
Comment (rhelmer) [inline public/index.html:13]: Did you mean to leave this commented? Should it just be removed if so?

File: src/routes/terms-of-service/CallToAction.svelte [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  $: if (scrollY > 20) {
+    showArrow = false;
+  } else {
+    showArrow = true;
+  }
+
+  // let mounted = false;
+  // onMount(() => {
+  //   mounted = true;
+  // });
```
Comment (rhelmer) [inline src/routes/terms-of-service/CallToAction.svelte:21]: Same question re: commented code :) If it serves a purpose (for testing etc) then we should have a comment above it at least (or do it in different way preferably)

File: src/routes/terms-of-service/Content.svelte [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+
+  <p>
+    Some sort of introduction lorem ipsum dolor sit amet, consectetur adipiscing
+    elit, sed do eiusmod tempor incididunt ut.
+  </p>
+
+  <hr />
+
+  <p>
+    Pioneer is an experimental initiative led by Mozilla to better understand
```
Comment (rhelmer) [inline src/routes/terms-of-service/Content.svelte:54]: Should we just `s/Pioneer/Ion` on this page?

**Review by rhelmer — CHANGES_REQUESTED** [review]
The onboarding storybook looks great!

One thing I noticed is that the study card looks a little off, missing CSS?

image

File: src/routes/terms-of-service/Content.svelte (hunk 3 above)
Comment (hamilton) [reply src/routes/terms-of-service/Content.svelte:54]: Yes, this copy will change in the very near future anyway.

**Comment by hamilton** [comment]
@rhelmer thanks for the review! The issue with that study card story is I'm importing the _new_ css and not the old stuff. I've decided to just remove the study card story for now since it will be replaced before we even touch the current study card implementation.

**Comment by hamilton** [comment]
@rhelmer Made the changes described above. I've done a pass on this with Ryan as well and have addressed his feedback. I think this is ready to merge.

**Pull request merged**